plugins/modules/zos_apf.py

```diff
@@ -546,6 +549,7 @@ def main():
         if batch:
             for item in batch:
                 item['opt'] = opt
+                # item['dsname'] = DataSet.escape_data_set_name(item.get('library'))
```

Comment:
we can remove this comment

Reply:
is a small change but will avoid confusion when working on this module in the future

Reply:
agreed